ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/services/reset/ResetServiceDatabaseIT.scala

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.services.reset
+
+import java.sql.{Connection, DriverManager}
+
+import anorm.SqlParser._
+import anorm.{SQL, SqlStringInterpolation}
+import com.digitalasset.ledger.api.testing.utils.MockMessages
+import com.digitalasset.platform.sandbox.services.reset.ResetServiceDatabaseIT.countRowsOfAllTables
+import com.digitalasset.platform.sandbox.services.{DbInfo, SandboxFixture}
+import com.digitalasset.platform.store.DbType
+import com.digitalasset.resources.ResourceOwner
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Try
+
+abstract class ResetServiceDatabaseIT extends ResetServiceITBase with SandboxFixture {
+
+  "ResetService" when {
+
+    "run against a database backend" should {
+
+      "leave the tables in the expected state" in {
+
+        val ignored = Set(
+          "flyway_schema_history", // this is not touched by resets, it's used for migrations
+          "packages" // preserved by the reset to match the compiled packages still loaded in the engine
+        )
+
+        for {
+          ledgerId <- fetchLedgerId()
+          party <- allocateParty(MockMessages.party)
+          _ <- submitAndExpectCompletions(ledgerId, 10, party)
+          _ <- reset(ledgerId)
+          counts <- countRowsOfAllTables(ignored, database.get)
+        } yield {
+
+          val expectedToHaveOneItem = Set(
+            "parameters", // a new set of parameters is stored at startup
+            "participant_command_completions", // using static time, one checkpoint is going to be saved
+            "ledger_entries" // using static time, one checkpoint is going to be saved
+          )
+
+          for ((table, count) <- counts if expectedToHaveOneItem(table)) {
+            withClue(s"$table has $count item(s): ") {
+              count shouldBe 1
+            }
+          }
+
+          // FIXME this appears to be racy, forcing us to make a loose check
+          val expectedToHaveOneItemOrLess = Set(
+            "configuration_entries"
+          )
+
+          for ((table, count) <- counts if expectedToHaveOneItemOrLess(table)) {
+            withClue(s"$table has $count item(s): ") {
+              count should be <= 1
+            }
+          }
```

Comment:
This is caused by some apparent race condition in the initialization with regards to `configuration_entries`, opening a ticket to track this.